tests/test_meshtastic_utils_message_paths.py

```
@@ -481,3 +552,210 @@ def test_on_meshtastic_message_non_text_plugin_exception(reset_meshtastic_global
         on_meshtastic_message(packet, _make_interface())
 
     mock_logger.exception.assert_any_call("Plugin %s failed", "boom")
+
+
+def _base_config_with_routing(chat_portnums=None, disabled_portnums=None):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_base_config_with_routing`

(ANN202)